src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,55 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+const NO_PARENTS = '-';
+const FEMALE = 'f';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const { name, sex, born, died, fatherName, motherName, mother, father } =
+    person;
+  const female = sex === FEMALE;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({ 'has-background-warning': slug === person.slug })}
+    >
+      <td>
+        <Link
+          to={person.slug}
+          className={classNames({
+            'has-text-danger': female,
+          })}
+        >
+          {name}
+        </Link>
```

Comment:
- create the <PersonLink person={person} /> and use it for all existing people;
![image](https://github.com/mate-academy/react_people-table-basics/assets/63243136/94cf25ee-7351-486f-9284-5580dfea2fe6)